src/base.jl

```diff
@@ -698,17 +755,16 @@ end
 Return a TimeSeries.TimeArray where the forecast data has been multiplied by the forecasted
 component field.
 """
-function get_forecast_values(component::Component, forecast::Forecast)
+function get_forecast_values(component::Component, forecast::IS.Forecast)
     return IS.get_forecast_values(PowerSystems, component, forecast)
 end
 
 """
-    get_forecast_initial_times(sys::System)::Vector{Dates.DateTime}
+    get_forecast_initial_times(sys::System) -> Vector{Dates.DateTime}
```

Comment:
Is the `->` documented somewhere as a Julia rule?

Reply:
![Screen Shot 2019-11-21 at 4 47 59 PM](https://user-images.githubusercontent.com/1813121/69386097-d0664f80-0c7e-11ea-9163-53be57d8fe8f.png)

Reply:
I added the use of DocStringExtensions in https://github.com/NREL/PowerSystems.jl/pull/390/commits/a58e6eca0d257a8a83ca01ef4bf106e9d39ae400 and https://github.com/NREL/PowerSystems.jl/pull/390/commits/2d9d2e023264c5ad02c5efec7e64b4e08b0a5842. See above screenshot for snippet of code in the file versus what shows up in the REPL when requesting for documentation. The same docstring that shows up in the REPL will show up in the documentation. 

We minimize the code we have to repeat in the function signature and documentation, however we lose some control on how the docstring is presented.